test/sandbox-test.js

```diff
@@ -243,6 +243,28 @@ describe("sinonSandbox", function () {
         }
     });
 
+    describe(".restore", function () {
+        it("throws when passed arguments", function () {
+            this.sandbox = sinonSandbox.create();
+
+            var failed = false;
```

Comment:
Close. You can use [assert.threw](http://docs.busterjs.org/en/latest/modules/referee/#threw) or optionally [assert.exception](http://docs.busterjs.org/en/latest/modules/referee/#exception) (for callbacks).